MAINTAINERS.md

```diff
@@ -4,8 +4,8 @@ The following table lists the names and email addresses of project maintainers.
 
 | Name | Email |
 | ---- | ----- |
-| Anil Vishnoi | avishnoi@redhat.com |
-| Brent Salisbury | bsalisbu@redhat.com |
-| Dave Tucker | dave@dtucker.co.uk |
-| Hiram Chirino | hiram@hiramchirino.com |
-| Russell Bryant | russell.bryant@gmail.com |
\ No newline at end of file
+| Anil Vishnoi | `avishnoi@redhat.com` |
+| Brent Salisbury | `bsalisbu@redhat.com` |
+| Dave Tucker | `dave@dtucker.co.uk` |
+| Hiram Chirino | `hiram@hiramchirino.com` |
+| Russell Bryant | `russell.bryant@gmail.com` |
```

Comment:
This is weird. I wasn't able to reproduce it.

Reply:
I wasnt able to locally either but assumed it was in today's updates, here is the CI job:
https://github.com/nexodus-io/nexodus/actions/runs/4854970441